src/main/java/edu/cornell/kfs/module/purap/web/struts/JadgaerAdminB2BAction.java

```diff
@@ -0,0 +1,43 @@
+package edu.cornell.kfs.module.purap.web.struts;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.struts.action.ActionForm;
+import org.apache.struts.action.ActionForward;
+import org.apache.struts.action.ActionMapping;
+import org.kuali.kfs.kns.web.struts.action.KualiAction;
+import org.kuali.kfs.krad.util.GlobalVariables;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.kfs.module.purap.exception.B2BConnectionException;
+import org.kuali.kfs.module.purap.web.struts.B2BForm;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.context.SpringContext;
+
+import edu.cornell.kfs.module.purap.CUPurapConstants.JaggaerRoleSet;
+import edu.cornell.kfs.module.purap.document.service.CuB2BShoppingService;
+
+public class JadgaerAdminB2BAction extends KualiAction {
```

Comment:
Minor typo in classname. Fixing this will also require adjusting any related config or SQL files.

Reply:
uggg, good notice .... switchign to JaggaerAdminB2BAction